### DISCUSSION TITLE
How to make roborock S1 speak english?

### DISCUSSION THREAD

**Discussion by dansity** [Hypfer/Valetudo#1772] [Q&A / Support] [answered]
I have installed valetudo on my robot. It's an oldie but does its job.
Now it speaks chinese. There is no roborock.conf under /mnt/default
I could not find any url to install voicepack those are seemingly for newer robots.

Any help appreciated.

**Comment by Hypfer** [comment, marked as answer]
https://builder.dontvacuum.me/pkg/voice/rockrobo/voice-v1/

although EN should be inbuilt into the firmware, meaning that you should be able to just put in any URL, any hash and the language code EN to switch to that

**Reply by dansity** [reply]
Thank you, worked.
